src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -385,20 +378,17 @@ def getCEStatus(self):
     #############################################################################
 
     def _renewJobs(self, jobList):
-        """Written for the REST interface - jobList is already in the REST format
+        """Written for the REST interface - jobList is already in the ARC format
```

Comment:
I think this comment is probably incorrect, as the old ARC format involved the protocol (gsiftp) also. The new ARC - REST format has just the job ID.